### PR TITLE
Refs #20049 - Retry various integration tests

### PR DESCRIPTION
**Pull request by dLobatog**
New breakages found that could be fixed by retrying (timeouts, DNS
errors...)

### CODE REVIEW

**Comment by mmoll** [comment]
[test upgrade]

**Comment by dLobatog** [comment]
@mmoll greener than a Prius

File: test/integration/audit_js_test.rb [hunk 1]
```diff
@@ -1,6 +1,8 @@
 require 'integration_test_helper'
 
 class AuditJSTest < IntegrationTestWithJavascript
+  # intermittent failures:
+  # AuditJSTest.test_0001_show audit with diff
   test "show audit with diff" do
```
Comment (tbrisker) [inline test/integration/audit_js_test.rb]: shouldn't this (and others below) `extend Minitest::OptionalRetry` ?

Comment (dLobatog) [reply test/integration/audit_js_test.rb]: Indeed...

**Comment by tbrisker** [comment]
Thanks @dLobatog !

**Pull request merged**